docs/sphinx/developers/ReadPhysicalSize.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * #%L
+ * OME Bio-Formats package for reading and converting biological file formats.
+ * %%
+ * Copyright (C) 2005 - 2016 Open Microscopy Environment:
+ *   - Board of Regents of the University of Wisconsin-Madison
+ *   - Glencoe Software, Inc.
+ *   - University of Dundee
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+import java.io.IOException;
+
+import loci.common.services.DependencyException;
+import loci.common.services.ServiceException;
+import loci.common.services.ServiceFactory;
+import loci.formats.FormatException;
+import loci.formats.FormatTools;
+import loci.formats.ImageReader;
+import loci.formats.ImageWriter;
+import loci.formats.MetadataTools;
+import loci.formats.meta.IMetadata;
+import loci.formats.services.OMEXMLService;
+import ome.units.UNITS;
+import ome.units.quantity.Length;
+import ome.units.unit.Unit;
+import ome.xml.model.enums.DimensionOrder;
+import ome.xml.model.enums.EnumerationException;
+import ome.xml.model.enums.PixelType;
+import ome.xml.model.primitives.PositiveInteger;
+
+/**
+ * Example class that shows how to read and convert the physical X, Y and Z dimensions of a file
+ * Bio-Formats version 5.2 or later.
+ */
+public class ReadPhysicalSize {
+    
+    /**
+     * Construct a new ReadPhysicalSize that will read and convert the physical dimensions of a file
+     *
+     * @param inputFile the file to be read
+     */
+    public ReadPhysicalSize(String inputFile) {
+        ImageReader reader = new ImageReader();
+        IMetadata omeMeta = MetadataTools.createOMEXMLMetadata();
+        reader.setMetadataStore(omeMeta);
+        reader.setId(inputFile);
+        
+        Unit<Length> targetUnit = UNITS.MICROMETER;
+        
+        for (int series=0; series<reader.getSeriesCount(); series++) {
```

Comment:
The getSeriesCount and getImageCount are certainly returning different values, ie you could have 5 series with each series having 10 images etc. Though having thought about it, it really doesn't make any sense to check the physical size on each image, these are going to be fixed values based on the hardware used rather than settings that will change during acquisition. Perhaps, as you had suggested earlier, a commented note to say its being read from the first image would make more sense?